components/Common/ThemeToggle/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { MoonIcon, SunIcon } from '@heroicons/react/24/outline';
+import { AccessibleIcon } from '@radix-ui/react-accessible-icon';
+import { useTheme } from 'next-themes';
+import type { FC } from 'react';
+import { useIntl } from 'react-intl';
+
+import styles from './index.module.css';
+
+const ThemeToggle: FC = () => {
+  const intl = useIntl();
+  const { theme, setTheme } = useTheme();
+
+  const isDark = theme === 'dark';
+
+  const toggleTheme = (isKeyPress?: boolean) => {
+    if (isKeyPress) {
```

Comment:
Why?

Reply:
> This is the final port of the https://github.com/nodejs/nodejs.org/pull/5730. Much of the functionality is the same to end users, but with different tech.

So I have base ma work on "old" themeToggle. So why not remove it. I will test when we focus on button en press enter if it's toggle

Reply:
Right, but that just means that the underlying technology is still `next-theme` but the base component should be simple.

Reply:
OK go to remove that 👍